miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -143,12 +156,17 @@ end
 
 #! Increments the account nonce by the provided value.
 #!
+#! Panics if
+#!   - the invocation of this procedure does not originate from the native account.
+#!   - the value is greater than 2^32 - 1
```

Comment:
We should probably move "panics" section to the end of the doc comments (similar to how we do it in Rust). But that's for another PR.